awx/ui-next/Makefile

```diff
@@ -0,0 +1,87 @@
+ui_next_mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+ui_next_dir := $(dir $(ui_next_mkfile_path))
+
+ifeq ($(current_dir), undefined)
+	ui_next_dir := $(dir $(ui_next_mkfile_path))
+else
+	ui_next_dir := $(shell realpath --relative-to $(current_dir) $(ui_next_dir))/
+endif
+
+ANSIBLE_UI_GIT_REPO_HTTPS ?= https://github.com/ansible/ansible-ui.git
```

Comment:
These will always be set. The double-edged sword of conditional assignment :). It does mean that those `if -z` conditionals in the commands below are useless unless someone shadows these somewhere (either by {un,}setting them at the top of the rule or from the env or command line. I don't love this because it means you probably can't just run `make` without setting/unsetting them.